realm-jni/src/io_realm_internal_SharedGroup.cpp

```diff
@@ -295,3 +304,23 @@ JNIEXPORT jboolean JNICALL Java_io_realm_internal_SharedGroup_nativeCompact(
     CATCH_STD()
     return false;
 }
+
+JNIEXPORT jlongArray JNICALL Java_io_realm_internal_SharedGroup_nativeGetVersionID
+        (JNIEnv *env, jobject, jlong native_ptr)
+{
+    TR_ENTER()
+    SharedGroup::VersionID version_id = SG(native_ptr)->get_version_of_current_transaction();
+
+    jlong version_array [2];
+    version_array[0] = static_cast<jlong>(version_id.version);
```

Comment:
I don't think it matter because we only compare it to other versions coming from JNI, so even if there casting issues, the comparison would still work?